zebra-chain/src/notes/sprout/nullifiers.rs

```diff
@@ -0,0 +1,101 @@
+//!
```

Comment:
```suggestion
```
If the goal is to suppress missing doc warnings, using `#![allow(missing_doc)]` is always better because empty doc comments are unlintable.